docs/usage/configuration/config_documentation.md

```diff
@@ -3670,14 +3671,63 @@ opentracing:
         false
 ```
 ---
-## Workers ##
+## Worker configuration for main process
 Configuration options related to workers.
+Workers are used to scale horizontally and distribute the load to different processes.
+
+The worker configuration is divided into two parts.
+
+1. These switches for the shared configuration to give the main process
+   the necessary information.
+1. And the [switches for the worker configuration file](#worker-configuration-for-workers-configuration)
+   to configure each worker them self are futher below.
+
+The manual how to configure workers is described in detail in the
+[worker documentation](../../workers.md)
+
+---
+### `worker_replication_secret`
+
+A shared secret used by the replication APIs to authenticate HTTP requests
+from workers.
+
+By default this is unused and traffic is not authenticated.
+
+Example configuration:
+```yaml
+worker_replication_secret: "secret_secret"
+```
+---
+### `start_pushers`
+
+Controls sending of push notifications on the main process. Set to `false`
+if using a [pusher worker](../../workers.md#synapseapppusher). Defaults to `true`.
+
+Example configuration:
+```yaml
+start_pushers: false
+```
+---
+### `pusher_instances`
+
+It is possible to run multiple [pusher workers](../../workers.md#synapseapppusher),
+in which case the work is balanced across them. Use this setting to list the pushers by
+[`worker_name`](#worker_name).
+
+If only one pusher worker is configured, this setting is not necessary.
```

Comment:
Can we also say something like this?

```
If no pusher workers are configured, this setting is not necessary---but
[`start_pushers`](#start_pushers) should be set to `true` so that the
main process will send out push notifications.
```

Reply:
[This](https://github.com/matrix-org/synapse/blob/develop/synapse/config/workers.py#L172) logic says that method is outdated. See [efforts](https://github.com/matrix-org/synapse/pull/14294) to change this.

Reply:
The comment (for the `federation_sender`) in the code is interesting. I try to document the current status here. If the logic has changed, it needs a new PR to update the documentation, deprecate the old federation_sender and remove the old code.

Reply:
> I think, something like this is better

Sure, but let's tweak the last sentence to say: "if you do not disable it by setting ...".

Reply:
=> https://github.com/matrix-org/synapse/pull/14086/commits/5045ca0edf89f691ca009fbac3d44194ddb22fe6